docs/learning/howto/use-roi-metrics.md

```diff
@@ -0,0 +1,202 @@
+# Using ROI Metric Data
+
+**ROI Metrics Data** stores arbitrary key value pairs against your job to help understand your ROI per job execution. The metrics will be stored against the each execution of the job, and accessible via API for further analysis.  This article will walk through a setup scenario and provide some helpful example scripts to gather data from the API.
+
+## Configuring the Metrics
+
+### Example Scenario:
+
+Customer X has decided to track how often time is saved with their newly implemented Automation job (uniquely titled `ROI Job`).  They know from historical issues that doing the same work prior to writing this job the manual steps took around 75 minutes across all teams, start to finish.  The job now runs in just seconds.  To configure the job follow the steps or download the job definition.
```

Comment:
Given that they're measuring _total_ time, let's replace "how often" with "how much".

Second sentence should read: They know from historical ticket-data that doing the same work manually - prior to writing this job - took around 75 minutes across all teams, start to finish.